plugins/modules/zypper.py

```diff
@@ -324,10 +324,11 @@ def parse_zypper_xml(m, cmd, fail_not_found=True, packages=None):
             m.fail_json(msg=errmsg, rc=rc, stdout=stdout, stderr=stderr, cmd=cmd)
         else:
             return {}, rc, stdout, stderr
-    elif rc in [0, 106, 103]:
+    elif rc in [0, 102, 103, 106]:
```

Comment:
What is 106?

Reply:
From the zypper man page (https://en.opensuse.org/SDB:Zypper_manual_(plain))
```
       106 - ZYPPER_EXIT_INF_REPOS_SKIPPED
           Some repository had to be disabled temporarily because it failed to refresh. You should check your
           repository configuration (e.g. zypper ref -f).
```
But I didn't touch that one ...

Reply:
Ah sorry, I totally missed that it was there before, just not at the end... Thanks :)